pallets/sto/src/lib.rs

```diff
@@ -107,12 +127,18 @@ impl<Balance: Clone> Into<FundraiserTier<Balance>> for PriceTier<Balance> {
     fn into(self) -> FundraiserTier<Balance> {
         FundraiserTier {
             total: self.total.clone(),
-            price: self.price.clone(),
-            remaining: self.total.clone(),
+            price: self.price,
+            remaining: self.total,
         }
     }
 }
 
+/// Wrapper type for Fundraiser name
+#[derive(
+    Decode, Encode, Clone, Debug, Default, PartialEq, Eq, PartialOrd, Ord, VecU8StrongTyped,
+)]
```

Comment:
```suggestion
#[derive(
    Decode, Encode, Clone, Default, PartialEq, Eq, PartialOrd, Ord, VecU8StrongTyped,
)]
#[cfg_attr(feature = "std", derive(Debug))]
```
To save space in the WASM binary.

Reply:
Have we actually tested if this helps? One would expect the optimizer to automatically strip unused code.

Reply:
I wouldn't be as confident re. the optimizer.. ;) At minimum this saves compile-time work post macro-expansion, skipping nameres, typeck, borrowck, codegen, llvm passes, etc...

Reply:
Assuming it saves 0.002 seconds and requires 30 seconds to implement the change, we'll need to compile 15,000 times to break even :)

Reply:
Anyway, I'm compiling it right now with the change and will do it without the change next to see the difference. I am more curious than anything.

Reply:
```
 error[E0277]: `FundraiserName` doesn't implement `core::fmt::Debug`
```

Reply:
38,50,831 bytes VS 38,50,831 bytes, with and without https://github.com/PolymathNetwork/Polymesh/pull/848/commits/129cca5f430273fab939cb887eed3c914750be2c